### PR TITLE
WIP: Basic structure for showcasing community contributions

### PR DESCRIPTION
**Pull request by cwognum**
Checklist:

- [X] Was this PR discussed in a issue? It is recommended to first discuss a new feature and let the community know whether you are planning or have started working on it before opening a PR.
- [ ] Add tests to cover the fixed bug(s) or the new introduced feature(s) (if appropriate).
- [X] Update the API documentation if a new function is added or an existing one is deleted.
- [x] Added a `news` entry.
  - _copy `news/TEMPLATE.rst` to `news/my-feature-or-branch.rst`) and edit it._

---

ping @maclandrol. I proposed a basic structure so we can discuss! What do you think?

### CODE REVIEW

**Comment by cwognum** [comment]
For now, I was thinking of having a table with a row per contribution. This will not scale, but is a sensible solution for the short-term.

File: docs/community/get_involved.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Get involved
+Whether you are a first-time contributor or open-source veteran, we welcome any contribution to the `molfeat` package. 
+Your contributions help to make sure this package can remain free-to-use and open-source.
+
+## Contribute
+The most straight-forward way to contribute is through Github. In short, you can help by: 
+
+- :star: **Star** the repository on Github
```
Comment (maclandrol) [inline docs/community/get_involved.md:8]: Add answering questions

File: docs/community/get_involved.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- :bug: **Report** bugs or **request** new features. 
+- :books: Improve **existing documentation** or add **new tutorials**.
+
+To learn more about this, please see the [Developers documentation](../developers/contribute.md).
+
+
+## Slack
+[![slack](https://img.shields.io/badge/Join%20our%20Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://join.slack.com/t/m2d2group/shared_invite/zt-16w1rjqqs-n81TiK~iB23XbZ0QWMYs~A)
+
+`molfeat` is an initiative that originated in the [M2D2 community](m2d2.io). To orchestrate these efforts the `#molfeat` channel
```
Comment (maclandrol) [inline docs/community/get_involved.md:19]: Instead, the molfeat community is part of the m2d2 community

File: docs/developers/contribute.md [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+- :star: **Star** the repository on Github
+- :computer: **Fix** existing code issues. 
+- :bug: **Report** bugs or **request** new features. 
+- :books: Improve **existing documentation** or add **new tutorials**.
 
 However you choose to contribute, please be mindful and respect our [code of conduct](https://github.com/datamol-io/molfeat/blob/main/.github/CODE_OF_CONDUCT.md).
 
+!!! note annotate "Get inspired by community contributions"
+    As we love community contributions, we decided to create some dedicate space in our documentation to highlight and celebrate such contributions.
+    Visit [Community Contribitons](../community/contributions.md) to learn more!s
```
Comment (maclandrol) [inline docs/developers/contribute.md:14]: There is a typo at the end.

File: docs/developers/contribute.md [hunk 4]
```diff
@@ -1,9 +1,17 @@
 # Contribute
 
-Welcome to the molfeat community! We appreciate everyone's contribution and welcome all. Apart from code contributions, there are various other ways to support the community such as answering questions, providing assistance to others, enhancing the documentation, giving a ⭐️ or sharing the project.
+Welcome to the molfeat community! We appreciate everyone's contribution and welcome all. Apart from code contributions, there are various other ways to support the community. In short, you can: 
+
+- :star: **Star** the repository on Github
```
Comment (maclandrol) [inline docs/developers/contribute.md:5]: This seems like a duplicate of the *ways to contribute section* below. Would likely make sense to pull that section up here and/or rephrase both a bit

File: README.md [hunk 5]
```diff
@@ -110,9 +110,11 @@ model_card.usage()
 
 Please cite Molfeat if you use it in your research: [![DOI](https://zenodo.org/badge/613548667.svg)](https://zenodo.org/badge/latestdoi/613548667).
 
-## Contribute
+## Contribute and Community
```
Comment (maclandrol) [inline README.md:113]: Create subsections  maybe ?  Or just contribute is enough

File: README.md [hunk 6]
```diff
@@ -110,9 +110,11 @@ model_card.usage()
...
 
-See [developers](docs/developers/) for a comprehensive guide on how to contribute to `Molfeat`
+See [developers](docs/developers/) for a comprehensive guide on how to contribute to `molfeat`. `molfeat` is a community-led
+initiative and whether you're a first-time contributor or an open-source veteran, this project greatly benefits from your contributions.
+To learn more about the community that is behind molfeat, please see [community](docs/community/).
```
Comment (maclandrol) [inline README.md:117]: Community and ecosystem of tools ? Maybe in community we can say things like if you like molfeat, you will like datamol and xyz ?

File: docs/community/contributions.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+## Plugins
+
+## Tutorials
+
+## Code
+<a href="https://github.com/datamol-io/molfeat/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=datamol-io/molfeat" />
+</a>
+
+Made with [contrib.rocks](https://contrib.rocks).
```
Comment (maclandrol) [inline docs/community/contributions.md:11]: You can remove the contrib rocks part, it's fine.

File: docs/community/contributions.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
+# Community contributions
+## Plugins
+
+## Tutorials
```
Comment (maclandrol) [inline docs/community/contributions.md:4]: I like the split

**Review by maclandrol — CHANGES_REQUESTED** [review]

**Comment by maclandrol** [comment]
> For now, I was thinking of having a table with a row per contribution that links to an external page (e.g. Github of a plugin, blog post, etc.). Or would you prefer copying as much as possible over to molfeat instead of having a link?

No, the table is perfect. We can also add some links or some badges in the original tutorials too.

**Comment by maclandrol** [comment]
Also ping @zhu0619

**Comment by cwognum** [comment]
@maclandrol I addressed your comments! The community contributions page probably still needs some context descriptions, but I'll do those later. In the meantime, could you add the IPU badges? I cannot see the raw README and thus cannot copy the IPU badge (I saw you edited the description, so you probably can?).

**Comment by cwognum** [comment]
**Reference in existing documentation:**

image

**New community contributions table**

image

**Review by maclandrol — APPROVED** [review]
LGTM, thanks @cwognum. 
Can be merged !

**Pull request merged**